### ISSUE TITLE
Merge with Moppy 1?

### ISSUE THREAD

**Issue by dmadison** [SammyIAm/Moppy2#17]
With all of the brouhaha over what to do with [the old repository](https://github.com/SammyIAm/Moppy) and how to get users to switch over to the 'new' version... have you considered ditching the "Moppy2" repo and just merging the history into the original Moppy repo? I'm not a git expert, but I believe it's possible to do.

**Comment by Lothean** [comment]
Nah, it would make using the original Moppy very casse couilles and I think it is better to never forget the history of Moppy!

**Comment by dmadison** [comment]
I don't see how it would. At the moment because of commit [b1e55ad](/SammyIAm/Moppy/commit/b1e55ad746fbf03e43671cb47a6a751325ed74c9), to use the original Moppy you would need to go to the commit history and browse the repo at a given commit (you commented on [that issue](/SammyIAm/Moppy/issues/203) as well). Adding additional commits wouldn't change that - although to make it easier you could create a Moppy 1.0 release.

You also wouldn't erase any history. You would append the Moppy 2.0 commits on top of 1.0. This would clear out the files from the repo, but the entire commit, issue, and pull request history of 1.0 would still be retained. If anything, splitting the repository at the version number obscures the history even more.

---

The upside is that you get the entire history of the software to the very beginning in one place. You get a centralized repository for everything with no confusion over which "version" to use. All existing links across the entire Internet will point to the most recent version, and you still have easy reference to past issues, pull requests, and discussions about the future of the software.

The downside is that you would lose the issues and pull requests from 2.0, and you would temporarily disrupt anyone working on improvements from a fork (although these can be transplanted as well). Which is why I'm raising this issue now at the repository's infancy, as this becomes more difficult as time goes on.

**Comment by SammyIAm** [comment]
Managed to get a redirect from `Moppy -> Moppy2`, so there's no longer any need for manual redirection.  I've rolled back the commit in the original repo that removed the code, and moved the whole repository over to [MoppyClassic](https://github.com/SammyIAm/MoppyClassic) just in case anyone's feeling nostalgic.

There should be no confusion about which version to use though: Moppy2 is the version that will continue to be supported and improved, and is already easier to use and troubleshoot than the original.

Apologies if the weird repo-shuffle has caused any discomfort-- I've learned for the next time this happens with a GitHub project.
